seed/static/seed/partials/inventory_detail.html

```diff
@@ -82,7 +82,8 @@ <h2>
                     </li>
 
                     <li role="menuitem">
-                        <a ng-click="open_match_merge_link_warning_modal(match_merge_link_record, 'manual')">Merge and Link Matches</a>
+                        <!-- ng-click="open_match_merge_link_warning_modal(match_merge_link_record, 'manual')" -->
+                        <a style="opacity:.5;">Merge and Link Matches</a>
```

Comment:
Opacity change is reasonable given we can't truly "disable" an anchor element 👍 

We could also override the cursor change with something like `cursor: not-allowed;` or `cursor: default;`:
```suggestion
                        <a style="opacity:.5; cursor: not-allowed;">Merge and Link Matches</a>
```

Reply:
Good call, I'll update and merge.  Thanks!